csharp/private/rules/resgen.bzl

```diff
@@ -0,0 +1,31 @@
+
+def _csharp_resx_impl(ctx):
+    print("todo")
+
+csharp_resx = rule(
+    implementation = _csharp_resx_impl,

```

Comment:
I'm thinking we'll want to provide a custom struct as mentioned in #77, as the name of the output `.resources` file is determined by either the csproj file name or the `RootNamespace` property. We could add an `out` property for modifying the name of the csproj file (rather than relying solely on `identifier`)

Reply:
The struct containing a `identifier` property (or `out` for controlling output name) is not really related to the csproj, but to how the resources are consumed by [C# compiler](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/resource-compiler-option):

```
-resource:filename[,identifier[,accessibility-modifier]]
```

With parameters like that, I can pass in a resources file named `hello.look.at.this.name.resources`, but use the identifier `hello.Strings.resources`. This is what I was getting at in [this](https://github.com/Brightspace/rules_csharp/pull/93#discussion_r333149380) discussion